routers/users.js

```diff
@@ -114,6 +114,12 @@ userRouter.get('/s/:search',[param('search').isString({ min: 5, max: 100 })], as
 		if (!users) {
 			throw new Error(`Can not find a user by this username`);
 		}
+		return res.json(users);
```

Comment:
Remember here you return "users" which makes sense according to your code, but the route wants to find a single user. Now you return an array, it makes sense to just return an object.

Reply:
I will make changes accordingly. Thanks